java/HealthVault Mobile/HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -163,6 +199,12 @@ public void onListItemClick(ListView parent, View v, int position, long id) {
 				case 0:
 					intent = new Intent(MainActivity.this, WeightActivity.class);
 					break;
+				case 1:
```

Comment:
Github won't let me comment on the code on line 189 since it is not directly included in this PR, but there is another indentation issue with the } else { on line 189 (needs to be indented once more).